src/test/kotlin/uk/gov/dluhc/printapi/rds/mapper/CertificateToPrintRequestMapperTest.kt

```diff
@@ -0,0 +1,167 @@
+package uk.gov.dluhc.printapi.rds.mapper
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import org.springframework.test.util.ReflectionTestUtils
+import uk.gov.dluhc.printapi.database.entity.CertificateFormat
+import uk.gov.dluhc.printapi.database.entity.CertificateLanguage
+import uk.gov.dluhc.printapi.database.entity.SourceType
+import uk.gov.dluhc.printapi.database.entity.Status
+import uk.gov.dluhc.printapi.rds.entity.Certificate
+import uk.gov.dluhc.printapi.rds.entity.ElectoralRegistrationOffice
+import uk.gov.dluhc.printapi.rds.entity.PrintRequest
+import uk.gov.dluhc.printapi.rds.entity.PrintRequestStatus
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidApplicationReference
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidBatchId
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidLocalAuthorityName
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidRequestId
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidUserId
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidVacNumber
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomGssCode
+import uk.gov.dluhc.printapi.testsupport.testdata.rds.rdsDeliveryBuilder
+import uk.gov.dluhc.printapi.testsupport.testdata.rds.rdsElectoralRegistrationOfficeBuilder
+import uk.gov.dluhc.printapi.testsupport.testdata.zip.aPhotoArn
+import uk.gov.dluhc.printapi.testsupport.testdata.zip.aPhotoZipPath
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneOffset.UTC
+import java.util.stream.Stream
+
+class CertificateToPrintRequestMapperTest {
+
+    private lateinit var mapper: CertificateToPrintRequestMapperImpl
+
+    @BeforeEach
+    fun setup() {
+        mapper = CertificateToPrintRequestMapperImpl()
+        ReflectionTestUtils.setField(mapper, "instantMapper", InstantMapper())
+    }
```

Comment:
This should work

```
@Mock
    private lateinit var instantMapper: InstantMapper

@InjectMocks
    private val mapper = CertificateToPrintRequestMapperImpl()
```

Reply:
Yes - but I chose to use the real object instead of a mock